perf/types.tsx

```diff
@@ -1,6 +1,6 @@
 declare global {
   interface Window {
-    runMeasures: () => Promise<ProfilerMeasureCycle>
+    runMeasures: (filter?: string) => Promise<ProfilerMeasureCycle>
```

Comment:
later we might think about `config` arg as an object, but for now has selected the simplest approach